ivy/functional/frontends/jax/numpy/statistical.py

```diff
@@ -514,3 +516,29 @@ def quantile(
     return ivy.quantile(
         a, q, axis=axis, keepdims=keepdims, interpolation=method, out=out
     )
+
+
+@to_ivy_arrays_and_back
+@with_unsupported_dtypes(
+    {"0.4.14 and below": ("complex64", "complex128", "bfloat16", "bool", "float16")},
+    "jax",
+)
+def nanquantile(
+    a,
+    q,
+    axis=None,
+    out=None,
+    overwrite_input=False,
+    method="linear",
+    keepdims=False,
+    interpolation=None,
+):
+    if interpolation:
+        method = interpolation
+        warnings.warn(
+            "interpolation is deprecated name for the method keyword argument since"
+            " version 1.22.0 in JAX"
+        ) 
+    return ivy.nanquantile(
```

Comment:
don't forget to change this function, maybe you can use regular quantile after filtering the `NaN` values as a compositional implementation